src/Connector/ArrayConverter/StandardToFlat/CategoryProperty.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flagbit\Bundle\CategoryBundle\Connector\ArrayConverter\StandardToFlat;
+
+use Akeneo\Tool\Component\Connector\ArrayConverter\StandardToFlat\AbstractSimpleArrayConverter;
+
+use function sprintf;
+
+/**
+ * Array converter to flatten the array from {@link CategoryProperty::$properties}.
+ */
+class CategoryProperty extends AbstractSimpleArrayConverter
+{
+    /**
+     * @param array<string, mixed> $data
+     * @phpstan-param array<string, mixed> $convertedItem
+     * @phpstan-param array<string, mixed> $options
+     *
+     * @return array<string, mixed>
+     */
+    // phpcs:ignore
+    protected function convertProperty($property, $data, $convertedItem, $options): array
```

Comment:
No need for a `phpcs:ignore` here.

```suggestion
    /**
     * {@inheritdoc}
     *
     * @phpstan-param array<string, mixed> $convertedItem
     * @phpstan-param array<string, mixed> $options
     *
     * @return array<string, mixed>
     */
    protected function convertProperty($property, $data, $convertedItem, $options): array
```